apix-impl/src/main/java/eu/xenit/apix/SpringConfiguration.java

```diff
@@ -4,9 +4,7 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ComponentScan(basePackageClasses = {
-        Version.class,
-})
+@ComponentScan(basePackages = {"eu.xenit.apix.alfresco"})
```

Comment:
`Version.class` does not work here, the Alfresco container crashes with the following message:

> 2024-06-14T07:05:32.724339994Z org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'schema.create.core' defined in URL [jar:file:/usr/local/tomcat/webapps/alfresco/WEB-INF/lib/alfresco-repository-21.20.jar!/alfresco/dbscripts/db-schema-context.xml]: Could not resolve placeholder 'db.script.dialect' in value "classpath:alfresco/dbscripts/create/${db.script.dialect}/AlfrescoCreate-RepoTables.sql"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'db.script.dialect' in value "classpath:alfresco/dbscripts/create/${db.script.dialect}/AlfrescoCreate-RepoTables.sql"

Using `basePackages = {"eu.xenit.apix.alfresco"}` was the best replacement I could come up with.